siac/hostdbcmd.go

```diff
@@ -117,15 +120,30 @@ func hostdbcmd() {
 				uptimeRatio = float64(uptime) / float64(uptime+downtime)
 			}
 
+			scanHistStr := ""
+			displayScans := host.ScanHistory
+			if len(host.ScanHistory) > scanHistoryLen {
+				displayScans = host.ScanHistory[len(host.ScanHistory)-scanHistoryLen:]
+			}
+			for _, scan := range displayScans {
+				if scan.Success {
+					scanHistStr += "1"
+				} else {
+					scanHistStr += "0"
+				}
+			}
```

Comment:
The amount of time between scans is random, adding a timescale would be very hard.